cmake/features.cmake

```diff
@@ -144,7 +144,7 @@ ie_dependent_option (ENABLE_GPU_DEBUG_CAPS "enable GPU debug capabilities at run
 
 ie_dependent_option (ENABLE_CPU_DEBUG_CAPS "enable CPU debug capabilities at runtime" ON "ENABLE_DEBUG_CAPS" OFF)
 
-if(ANDROID OR WINDOWS_STORE OR (MSVC AND (ARM OR AARCH64)))
```

Comment:
do we need it for 2022.2? It's already CF
for master we have fixed this and protobuf readers are enabled unconditionally

Reply:
I see. Let's close PR.

Reply:
BTW, you can see it here https://github.com/openvinotoolkit/openvino/pull/12881

Reply:
Great! Thank you